code/game/objects/items/weapons/tools.dm

```diff
@@ -768,3 +768,49 @@
 		playsound(loc, 'sound/items/change_drill.ogg', 50, 1)
 	update_tool()
 	return TRUE
+
+/obj/item/steelwool
+	name = "steel wool"
+	desc = "Harvested from the finest NanoTrasen steel sheep."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "steel_wool"
+	attack_verb = list("lacerated", "garroted")
+	w_class = ITEMSIZE_SMALL
+	var/lit
+
+/obj/item/steelwool/isFlameSource()
+	return lit
+
+/obj/item/steelwool/attackby(obj/item/W, mob/user)
+	if(W.isFlameSource())
+		ignite(W, user)
+	else if(istype(W, /obj/item/cell))
+		var/obj/item/cell/S = W
+		if(S.charge)
+			ignite(W, user)
+		else
+			to_chat(user, SPAN_WARNING("The cell isn't charged!"))
+
+
+/obj/item/steelwool/proc/ignite(var/L, mob/user)
+	lit = TRUE
+	user.visible_message(SPAN_NOTICE("[user] ignites the steel wool with \the [L]."), SPAN_NOTICE("You ignite the steel wool."), SPAN_NOTICE("You hear a gentle flame crackling."))
+	playsound(get_turf(src), 'sound/items/flare.ogg', 50)
+	desc += " Watch your hands!"
+	icon_state = "burning_wool"
+	set_light(2, 2, LIGHT_COLOR_LAVA)
+	addtimer(CALLBACK(src, .proc/endburn, user), 120 SECONDS, TIMER_UNIQUE)
+
+/obj/item/steelwool/proc/endburn(mob/living/carbon/human/user)
+	visible_message(SPAN_NOTICE("The steel wool burns out."))
+	if(!user.gloves)
```

Comment:
Are there exactly zero ways to extinguish an already lit piece of steel wool? Or do we need to check if it is still lit here?

What if my lit piece of steel wool has been used as a fuse in the meantime?

Reply:
At the moment there aren't any ways to extinguish steel wool (as I can't really think of any ways to do it - steel wool still burns when wet and blowing on it just adds oxygen to the reaction. Maybe stamping on it works? I've never played with the stuff). Also, I actually forgot to add in a line that deletes the steel wool when it gets stuffed into a can, as it's supposed to 'ruin' it and make it unusable so I don't think that second point should become an issue. unless something goes badly wrong.